src/Microsoft.SqlTools.ServiceLayer/EditData/UpdateManagement/CellUpdate.cs

```diff
@@ -6,6 +6,7 @@
 using System;
 using System.Globalization;
 using System.Text.RegularExpressions;
+using Microsoft.SqlTools.Hosting.Contracts;
```

Comment:
Are these needed any more? I think with the undo of your rename you can undo these changes?